specification/deploymentmanager/data-plane/Microsoft.DeploymentManager/2017-11-01-preview/rollouts.json

```diff
@@ -0,0 +1,568 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2017-11-01-preview",
+        "title": "AzureDeploymentManagerClient",
+        "description": "Azure Deployment Manager (ADM) client performs rollout orchestration of services using Azure Deployment Manager service."
+    },
+    "host": "azureservicedeploy.msft.net",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/api/rollouts": {
+            "post": {
+                "tags": [
+                    "Rollouts"
+                ],
+                "operationId": "Rollouts_Create",
+                "x-ms-long-running-operation": true,
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "summary":"Creates a new rollout.",
+                "description":"This is an asynchronous operation and can be polled to completion using the rollout identifier returned by this operation.",
+                "parameters": [
+                    { 
+                        "name": "artifactSource", 
+                        "in": "body", 
+                        "required": true, 
+                        "schema": { 
+                            "$ref": "#/definitions/ArtifactSource" 
+                        }, 
+                        "description": "Source details of the artifacts that defines the rollout."
+                    },
+                    { 
+                        "$ref": "#/parameters/ApiVersionParameter" 
+                    }
+                ],
+                "responses": {
+                    "201": {
+                        "description": "Created",
+                        "schema": {
+                            "$ref": "#/definitions/Rollout"
+                            }
+                    },
+                    "default":{
+                        "description": "ADM error text describing failure reason. This text is set as Reason-Phrase per HTTP RFC2616"
+                    }
+                }
+            }
+        },
+        "/api/rollouts/{rolloutId}": {
+            "get": {
+                "tags": [
+                    "Rollouts"
+                ],
+                "operationId": "RolloutId_Get",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "summary":"Gets detailed information of an ongoing or completed rollout.",
+                "parameters": [
+                    {
+                        "name": "rolloutId",

```

Comment:
Please move parameter definitions to global parameter section.

Reply:
Do you mean every parameters in all API need to move to global section, or the common parameters?